pyproject.toml

```diff
@@ -97,7 +98,7 @@ lint = [
 ]
 test = [
     "pytest>=7.4.2",
-    "pytest-asyncio>=0.21.1",
+    "pytest-asyncio==0.21.1",
```

Comment:
The latest `pytest-asyncio` changes the behavior of how tests are collected resulting in errors for `test_beanie_factory.py`. So, it'll be pinned to this version until the model definitions are changed to accommodate for that. I felt this is probably best done in another PR.